test/functional/page_objects/dashboard_page.js

```diff
@@ -198,8 +198,13 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
       return await testSubjects.exists('createDashboardPromptButton');
     }
 
-    async clickListItemCheckbox() {
-      await testSubjects.click('dashboardListItemCheckbox');
+    async checkDashboardListingSelectAllCheckbox() {
+      const element = await testSubjects.find('checkboxSelectAll');
+      const isSelected = await element.isSelected();
+      if(!isSelected) {
```

Comment:
Super little nit-pick. I think we generally write this w/ a space after the if: `if (!isSelected) {` instead of `if(!isSelected) {`.